frontend/src/App.tsx

```diff
@@ -45,6 +54,30 @@ function App() {
 		}
 	}
 
+	const loadUserUsingHandle = async (handle: string) => {
```

Comment:
why do we need to make a duplicate function, the only difference I see is it takes handle as input.
You can just add handle as a parameter in the original function and update previous calls to pass the state.

Or there's a better way,

We can have 2 states, `handle` and `user`
the handle is updated on typing in input, but only on pressing enter/clicking on search do we update user using setUser
and then we can make a side effect on `user` using `useEffect` which calls `loadUser` upon that.

Reply:
hi just looking at this now, is there a reason this pull request is not merged?